### ISSUE TITLE
32 bit python issues

### ISSUE THREAD

**Issue by springmeyer** [mapbox/windows-builds#35]
This may be user error or user mis-understanding, but here's my experience try to get 32 bit builds going locally:

- ran `settings.bat TARGET_ARCH=32`
- hit error that I need python installed.
- downloaded 32 bit python 2.7.7 msi, ran installer
- ran `settings.bat TARGET_ARCH=32` again
- hit error that I needed setuptools installed from `https://pypi.python.org/pypi/setuptools#installation-instructions`
- tried running `(Invoke-WebRequest https://bootstrap.pypa.io/ez_setup.py).Content | python -` in a powershell terminal
-  hit this error:

![screen shot 2015-03-25 at 4 57 05 pm](https://cloud.githubusercontent.com/assets/20300/6837966/06840dc6-d310-11e4-88b6-8b572f37cd92.png)

**Comment by BergWerkGIS** [comment]
It's: `settings.bat "TARGET_ARCH=32"`
Notice the quotes!

I have never run the scripts from PowerShell before, maybe that's the problem.

I'm using `Cmder` as console http://gooseberrycreative.com/cmder/
Download the 23MB 7z package and extract.

**Comment by springmeyer** [comment]
> It's: settings.bat "TARGET_ARCH=32"

Yes, I used quotes, but missed them in this ticket

We've been over this before and my memory is bad: but currently we depend on a system python but also install our own versions: Maybe we could move to always using our own versions?

**Comment by BergWerkGIS** [comment]
> Maybe we could move to always using our own versions?

As far as I remember, we already tried that.
But `gyp` was the problem then, couldn't get it work with our own Python.
That's why it also has to be __32bit__ Python, otherwise gyp won't work.
Whereas boost needs, depending on arch being built, 32bit __and__ 64bit.

**Comment by springmeyer** [comment]
> That's why it also has to be 32bit Python, otherwise gyp won't work.

I've forgotten the details here, so thanks for narrowing it down to this. Hopefully we can get to the bottom of why that was and fix things so that gyp + 64 bit python works.

**Comment by BergWerkGIS** [comment]
@springmeyer I tried on three different machines (from within PowerShell) and couldn't reproduce your error.

Interesting error `ImportError: DLL load failed`.
So it seems that some DLLs were not found or the ones found where not the correct ones.

Was this on a fresh machine or one that you have used before?

Try deleting the `tmp-bin` before calling `settings.bat`.
I moved things around a bit when I created the package for the executables.
